### DISCUSSION TITLE
Quarto is rendering "--" to "–"

### DISCUSSION THREAD

**Discussion by ivansaul** [quarto-dev/quarto-cli#3828] [Q&A] [answered]
Helllo. "How can I make Quarto render '--' as two hyphens instead of '–'? I am seeing Quarto using the character '–' instead of '--' in some of my documents and I would like to know how I can fix this problem. Is there any configuration or function I can use to force Quarto to use two hyphens instead of the '–' character? Any help would be greatly appreciated."

example.qmd
Hello --World--

after render to html

example.html
Hello–World–

**Comment by dragonstyle** [comment, marked as answer]
The `--` is being converted to an en dash. You can disable this 'smart' behavior like so:

```
---
title: Hello World
from: markdown-smart
---

Hello--World--
```

Or you can just escape the dashes like so:

```
---
title: Hello World
---

Hello\-\-World--
```

HTH!

**Reply by ivansaul** [reply]
@dragonstyle Thanks for the help. I appreciate it a lot.

**Comment by mcanouil** [comment]
FYI, behaviour related to an issue with shortcodes: #3744

**Reply by cscheid** [reply]
Yes, but we want to fix the shortcode issue for all formats (by unescaping the parameters).

**Reply by mcanouil** [reply]
I know, it's only for reference for any users looking for this. 😉
